docs/features/software-templates/writing-custom-actions.md

```diff
@@ -151,27 +151,49 @@ and writing of template entity definitions.
 
 ### Adding a TemplateExample
 
-A TemplateExample is a predefined structure that can be used to create custom actions in your software templates. It
-serves as a blueprint for users to understand how to use a specific action and its fields as well as to ensure
-consistency and standardization across different custom actions.
+A TemplateExample is a way to document different ways that your custom action can be used. Once added it will be visible
+in your backstage instance under [/create/actions](https://demo.backstage.io/create/actions) path. You can have multiple
+examples for one action that can demonstrate different combinations of inputs and how to use them.
 
-#### Define a TemplateExample and add to your Custom Action
+#### Define TemplateExamples
+
+Below is a sample TemplateExample that is used for `publish:github`. The source code is available
+on [github](https://github.com/backstage/backstage/blob/master/plugins/scaffolder-backend-module-github/src/actions/github.examples.ts)
+and preview on [demo.backstage.io/create/actions](https://demo.backstage.io/create/actions)
```

Comment:
```suggestion
and preview on [demo.backstage.io/create/actions](https://demo.backstage.io/create/actions#publish-github)
```

You can actually deep link, I think that makes it easier to follow in this case too 👍